config/config.go

```diff
@@ -52,19 +50,14 @@ type (
 		DnsTable string `yaml:"DnsTable"`
 	}
 
-	DurationCfg struct {
-		DurationTimeScale string `yaml:"DurationTimeScale"`
-	}
-
 	ScanningCfg struct {
 		ScanThreshold int    `yaml:"ScanThreshold"`
 		ScanTable     string `yaml:"ScanTable"`
 	}
 
 	TBDCfg struct {
-		DefaultBucketSize       float64 `yaml:"DefaultBucketSize"`
```

Comment:
Default Bucket Size never influence any results in the old TBD code. It was used to build an unused histogram